lib/resources/collection/index.js

```diff
@@ -448,7 +448,7 @@ Collection.prototype.save = function (ctx, fn) {
 
   domain.changed =  function (property) {
```

Comment:
changedAny & changedAll can be in another PR. Then, we can keep this PR unchanged.